modules/gdscript/tests/scripts/analyzer/errors/construct_abstract_script.notest.gd

```diff
@@ -0,0 +1,2 @@
+@abstract
+class_name TestConstructAbstractScriptNotest
```

Comment:
The test class look too simple for a real abstract class.

Only mark a class as abstract will not cover a typical abstract class.
An abstract class has typical full implemented functions and also abstract function you need to implement in the inheriting class. 

What does an abstract function signature look like?
```
@abstract
class_name TestConstructAbstractScriptNotest extends Node:

func foo() -> String:
    return "foo"

# define a abstract function you need to implement by using the annotation
@abstract
func bar() ->String;

# or define a abstract function by using the key word
abstract func bar() ->String;
```

Reply:
This PR does not implement abstract functions, only abstract classes.

Reply:
But an abstract class without abstract function makes not much sense, right?